blueocean-dashboard/src/main/js/components/RunDetails.jsx

```diff
@@ -85,8 +85,11 @@ class RunDetails extends Component {
 
         const baseUrl = buildRunDetailsUrl(organization, name, branch, runId);
 
-        const currentRun = this.props.runs.filter(
-            (run) => run.id === runId && decodeURIComponent(run.pipeline) === branch)[0];
+        /* eslint-disable arrow-body-style */
```

Comment:
I find those one liners a real pain because I can't easily drop a `debugger` statement in there to debug. I can switch it back if you prefer.